pip_audit/cli.py

```diff
@@ -2,9 +2,80 @@
 Command-line entrypoints for `pip-audit`.
 """
 
+import argparse
+import enum
+import logging
+import os
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=os.environ.get("PIP_AUDIT_LOGLEVEL", "INFO").upper())
+
+
+@enum.unique
+class OutputFormat(str, enum.Enum):
+    """
+    Output formats supported by the `pip-audit` CLI.
+    """
+
+    Plain = "plain"
```

Comment:
Would be nice for this to mirror pip's existing format options, where the default is "columns" (I'm imagining we'll probably have some kind of columns in the default output)

Reply:
Done!